Sources/Sentry/Public/SentryHub.h

```diff
@@ -45,27 +45,29 @@ SENTRY_NO_INIT
 /**
  * Captures a transaction and sends it to Sentry.
  *
- * @param transaction The Transaction to send to Sentry.
+ * @param transaction The transaction to send to Sentry.
```

Comment:
L: our official docs `sentry-docs` always `PascalCase` names that are meant as features, which is the case of a `Transaction`, represents a feature and not merely the wording transaction itself.
wondering why would we do differently.

eg https://docs.sentry.io/product/releases/health/#crash
you can see the wording `Discover` and `Issues` using Pascal case.

I'm not against and fine with the changes, just wanted to understand the reasoning

Reply:
Here, I think we are not referring to the feature but instead to the class. So I think "transaction" is appropriate. @brustolin, the discussion here could be interesting for you as well.

Reply:
Understood